packages/astro/src/core/build/plugins/plugin-internals.ts

```diff
@@ -16,7 +16,6 @@ export function vitePluginInternals(input: Set<string>, internals: BuildInternal
 						// Except for these packages as they're not bundle-friendly. Users with strict package installations
 						// need to manually install these themselves if they use the related features.
 						external: [
-							'shiki', // For syntax highlighting
```

Comment:
`shikiji` can be bundled, so we don't have to externalize it, and the user don't have to manually install themselves if using pnpm.